src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/publicApi/model/subjectAccessRequest/SarBespokeCondition.kt

```diff
@@ -0,0 +1,10 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource.publicApi.model.subjectAccessRequest
+
+import io.swagger.v3.oas.annotations.media.Schema
+
+@Schema(description = "Describes a bespoke condition on a licence")
+data class SarBespokeCondition(
```

Comment:
Maybe it'd be better to get rid of this class and have a list of strings.
I think tihs would make rendering easier